CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## unreleased
 * BraintreeVenmo
   * Add additional error parsing for Venmo errors
+  * Throw cancelation specific error for `BTVenmoClient.tokenize()` (fixes #1085) 
```

Comment:
Should we call out more specifically that we no longer return `nil, nil` in the completion for cancel? It's possible merchants have logic built around that expectation so may be good to call it out more explicitly.

Reply:
Done!